src/store/login/login.thunks.js

```diff
@@ -87,6 +128,7 @@ function fetchWallet (walletName, accountData) {
         dispatch(loginActions.goToCreateAccountAuthStep(wallet))
       }
     } catch (error) {
+      console.log(error)
```

Comment:
I would use `console.error` instead, so it's rendered as a normal error in the console (i think that it's more readable for people helping us debugging)